### PR TITLE
Support seeking playback on RosbridgePlayer

### PR DESCRIPTION
**Pull request by WatanabeToshimitsu**
## What
- Support seeking playback on RosbridgePlayer
- Add sample panel

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/webviz/pull/3#" title="Author self-approved">WatanabeToshimitsu</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fwebviz).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/dataware-tools/webviz/blob/dataware-tools/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/webviz-core/src/players/RosbridgePlayer.js [hunk 1]
```diff
@@ -124,6 +135,46 @@ export default class RosbridgePlayer implements Player {
...
+    const targetService = new ROSLIB.Service({
+      ros: rosClient,
+      name: serviceName,
+      serviceType,
+    });
+    rosClient.getServices((services) => {
+      if (services.indexOf(serviceName) !== -1) {
+        targetService.callService(request, (result) => {
+          if (result.success) {
+            if (onSuccess) {
```
Comment (yusukefs) [inline packages/webviz-core/src/players/RosbridgePlayer.js:160]: 遅くなりすみません！
ここは `_isServiceBusy` を戻さなくても大丈夫でしょうか？
それ以外はLGTMです。

Comment (WatanabeToshimitsu) [reply packages/webviz-core/src/players/RosbridgePlayer.js:160]: そのままコピペしてますね....
こうなってる意図が分からなかったので，全てのパスで false とするよう修正しました

**Comment by WatanabeToshimitsu** [comment]
@yusukefs 
レビューの反映と，バグの修正も追加しました
[MTG で共有した，RawMessage パネルと SceneSelector パネルが上手く共存できない問題](https://www.notion.so/humandatawarelab/2021-12-13-Weekly-Meeting-0586619d338c49adbfbb2f097023be78#5153ead3e2b448f7b19d1d453577e32c)を解決しています

**Review by yusukefs — APPROVED** [review]
ありがとうございます！
LGTM

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/webviz/pull/3#" title="Author self-approved">WatanabeToshimitsu</a>*, *<a href="https://github.com/dataware-tools/webviz/pull/3#pullrequestreview-832197333" title="Approved">yusukefs</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fwebviz).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/dataware-tools/webviz/blob/dataware-tools/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**